pom.xml

```
@@ -70,6 +70,25 @@
 			<version>7.0.5.Final</version>
 			<type>pom</type>
 		</dependency>
+
+		<dependency>
+			<groupId>org.mapstruct</groupId>
+			<artifactId>mapstruct</artifactId>
+			<version>1.5.5.Final</version>
```

replace to properties
```suggestion
			<version>${mapstruct.version}</version>
```